packages/webapp/pages/onboarding.tsx

```diff
@@ -176,44 +184,58 @@ export function OnboardPage(): ReactElement {
+              You can always change theselater.
+            </p>
+          </>
         )}
-        <p className="px-6 mt-3 text-center whitespace-pre-line text-theme-label-secondary typo-body">
-          Pick a few subjects that interest you. You can always change these
-          later.
-        </p>
         <div className="flex flex-1" />
         {isFiltering ? (
-          <FilterOnboarding
-            className="grid-cols-2 tablet:grid-cols-4 laptop:grid-cols-6 mt-4"
-            onSelectedTopics={hasSelectedTopics}
-          />
+          <>
+            <FilterOnboarding
+              className="grid-cols-2 tablet:grid-cols-4 laptop:grid-cols-6 mt-4"
+              onSelectedTopics={hasSelectedTopics}
+            />
+            <div className="flex sticky bottom-0 z-3 flex-col items-center pt-4 mt-4 w-full">
+              <div className="flex absolute inset-0 -z-1 w-full h-1/2 bg-gradient-to-t to-transparent from-theme-bg-primary" />
+              <div className="flex absolute inset-0 top-1/2 -z-1 w-full h-1/2 bg-theme-bg-primary" />
+              <Button className="btn-primary w-[22.5rem]" onClick={onClickNext}>
+                Next
+              </Button>
+            </div>
+          </>
         ) : (
-          <img
-            alt="Sample illustration of selecting topics"
-            src={onboardingIntroduction}
-            className="absolute tablet:relative top-12 tablet:top-0 tablet:scale-125"
-          />
-        )}
-        <div className="flex sticky bottom-0 z-3 flex-col items-center pt-4 mt-4 w-full">
-          <div className="flex absolute inset-0 -z-1 w-full h-1/2 bg-gradient-to-t to-transparent from-theme-bg-primary" />
-          <div className="flex absolute inset-0 top-1/2 -z-1 w-full h-1/2 bg-theme-bg-primary" />
-          <Button className="btn-primary w-[22.5rem]" onClick={onClickNext}>
-            Next
-          </Button>
-          <MemberAlready
-            className={{
-              container: 'text-theme-label-tertiary py-4',
-              login: 'text-theme-label-primary',
-            }}
-            onLogin={() =>
-              setAuth({ isAuthenticating: true, isLoginFlow: true })
+          <div className="hidden laptop:block">
+            {
+              // eslint-disable-next-line jsx-a11y/media-has-caption
+              <video
+                loop
+                autoPlay
+                muted
+                className="absolute tablet:relative -top-14 -z-1 mt-1 -mb-16 tablet:scale-150"
+              >
+                <source
+                  src={cloudinary.onboarding.video.mp4}
+                  type="video/mp4"
+                />
+                <source
+                  src={cloudinary.onboarding.video.webm}
+                  type="video/webm"
+                />
+              </video>
             }
-          />
-        </div>
+            <p className="relative z-3 mb-6 text-center opacity-64 typo-callout text-theme-label-quaternary">
```

Comment:
We normally don't use opacity for text styling.
I do think design theme avoided our guidelines here

Reply:
Did not intend to use opacity, but to get the correct text colour

Reply:
It should simply be quarternary without the extra opacity according to Tsahi.
Think the design just got overwritten.

https://dailydotdev.slack.com/archives/C05P8U95VFT/p1694669630642729?thread_ts=1694611265.506569&cid=C05P8U95VFT

Reply:
Already changed it to it in 6e6d527